socketio/examples/callback.rs

```diff
@@ -35,9 +35,8 @@ fn main() {
         // Event specified by enum
         .on(Event::Close, |_, socket| {
             println!("Socket Closed");
-            socket
-                .emit("message", json!({"foo": "Hello server"}))
-                .expect("Error emitting");
+            let result = socket.emit("message", json!({"foo": "Hello server"}));
+            assert!(result.is_err());
```

Comment:
Why this change?

Reply:
`emit().expect()` will panic when `on_close` called which will not reconnect when server is interrupted.

Reply:
I see, but `assert!` is also resulting in a panic (see source [comment](https://doc.rust-lang.org/src/core/macros/mod.rs.html#1373-1376)). So where is the difference? The error message?

Reply:
em... you are right

Reply:
Could you please remove that change? :)

Reply:
Of course.